components/Comment/lib/commentTypes.tsx

```diff
@@ -0,0 +1,35 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faComments, faStar, faLayerGroup, faCaretDown, faCheck } from "@fortawesome/free-solid-svg-icons";
+import { COMMENT_TYPES } from "./types";
+
+const commentTypes = [
+  {
+    label: "Discuss",
+    verb: "discussing",
+    value: COMMENT_TYPES.DISCUSSION,
+    isDefault: true,
+    group: "contribute",
+    icon: <FontAwesomeIcon icon={faComments} />,
```

Comment:
@calvinhlee23 / @lightninglu10 As we discussed in Colorado, this is the right way to import font awesome icons. It is most performant according to their docs.